crates/wash-cli/src/call.rs

```diff
@@ -115,82 +127,61 @@ pub async fn handle_command(
         "calling component function over wRPC"
     );
 
-    // TODO: Reenable
-    //let nc = create_nats_client_from_opts(
-    //    &opts.rpc_host,
-    //    &opts.rpc_port,
-    //    opts.rpc_jwt.clone(),
-    //    opts.rpc_seed.clone(),
-    //    opts.rpc_credsfile.clone(),
-    //    opts.rpc_ca_file.clone(),
-    //)
-    //.await?;
-    //
-    //let mut headers = async_nats::HeaderMap::new();
-    //headers.insert("source-id", "wash");
-    //
-    //let lattice = opts
-    //    .lattice
-    //    .clone()
-    //    .unwrap_or_else(|| DEFAULT_LATTICE.to_string());
-    //
-    //// TODO: Configure invocation timeouts
-    //let wrpc_client = wasmcloud_core::wrpc::Client::new(
-    //    nc,
-    //    &lattice,
-    //    &component_id,
-    //    headers,
-    //    Duration::from_secs(10),
-    //);
-
-    //let (namespace, package, interface, name) = parse_wit_meta_from_operation(&function).context(
-    //    "Invalid function supplied. Must be in the form of `namespace:package/interface.function`",
-    //)?;
-    //let instance = format!("{namespace}:{package}/{interface}");
-    //let name = name.context(
-    //    "Invalid function supplied. Must be in the form of `namespace:package/interface.function`",
-    //)?;
-    //debug!(
-    //    ?component_id,
-    //    ?instance,
-    //    ?name,
-    //    ?lattice,
-    //    "invoking component"
-    //);
+    let lattice = opts
+        .lattice
+        .clone()
+        .unwrap_or_else(|| DEFAULT_LATTICE.to_string());
+    let prefix = Arc::new(format!("{}.{component_id}", &lattice));
+
+    let nc = create_client_from_opts_wrpc(&opts)
+        .await
+        .context("failed to create async nats client")?;
+    let wrpc_client = wrpc_transport_nats::Client::new(nc, prefix.as_str(), None);
+
+    let (namespace, package, interface, name) = parse_wit_meta_from_operation(&function).context(
```

Comment:
Just a note here that resource methods and statics are identified as `resource.method`, we need to make sure that whatever UI is used here supports this addressing scheme.
I feel like the CLI probably should separate the instance from function identifier for this to work well

Reply:
Ah could you make an issue for this? This is probably something that needed to be changed when resource support went in, and it would probably be quick and a great "first ticket" maybe for someone to handle.

Reply:
Nvm made an issue here:

https://github.com/wasmCloud/wasmCloud/issues/2572